typescript/src/bitcoin.ts

```diff
@@ -374,6 +481,27 @@ export function computeHash160(text: string): string {
   return hash160.digest(Buffer.from(text, "hex")).toString("hex")
 }
 
+/**
+ * Computes the double SHA256 for the given text.
+ * @param text - Text the double SHA256 is computed for.
+ * @returns Hash as a 32-byte un-prefixed hex string.
+ */
+export function computeHash256(text: string): string {
+  const firstHash = sha256.digest(Buffer.from(text, "hex"))
+  return sha256.digest(firstHash).toString("hex")
+}
+
+/**
+ * Converts a hash in hex string in little endian to a BigNumber.
+ * @param hash - Hash in hex-string format.
+ * @returns BigNumber representation of the hash.
+ */
+export function hashLEToBigNumber(hash: string): BigNumber {
+  return BigNumber.from(
+    "0x" + Buffer.from(hash, "hex").reverse().toString("hex")
```

Comment:
```suggestion
    Hex.from(hash, "hex").reverse().toString()
```

Reply:
Done in 072614a49f7d9c2f96b1d51f6f5f4cff2b8ae64d.